samples/xamarin-forms/Cinephile/Cinephile.ViewModels/AboutViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Reactive;
+using System.Reactive.Concurrency;
+using ReactiveUI;
+
+namespace Cinephile.ViewModels
+{
+    public class AboutViewModel : ViewModelBase
+    {
+        public AboutViewModel(IScheduler mainThreadScheduler = null, IScheduler taskPoolScheduler = null, IScreen hostScreen = null)
```

Comment:
Worth putting each dependency on a new line to prevent the horizontal scrollbar appearance when browsing source files on GitHub :smile_cat: 

Btw great set of updates, keep up! :1st_place_medal:

Reply:
Done :)

Thanks for the early reviews, keep shooting and I'll fix ;)
I appreciate it.